web/src/components/LocationLayout.tsx

```diff
@@ -73,7 +75,10 @@ const LocationLayout = (props: PropsType): ReactElement => {
       footer={!isLoading && showFooter ? <LocationFooter city={cityModel.code} language={languageCode} /> : null}
       modal={feedbackModal}
       toolbar={toolbar}>
-      {children}
+      <>
+        {children}
+        {buildConfig().featureFlags.developerFriendly && cityModel.name === 'München' && <RasaWidget />}
```

Comment:
Probably does not really matter but I'd prefer a check on `code` since it is unique and does not contain uppercase or special characters.
```suggestion
        {buildConfig().featureFlags.developerFriendly && cityModel.code === 'muenchen' && <RasaWidget />}
```